homework/content.js

```diff
@@ -1,37 +1,43 @@
 'use strict';
 
-(function() {
-  var _wrapper = document.querySelector('#note-content-wrapper');
+(function (exports){
 
-  function start() {
-    window.addEventListener('note-open', function(event) {
-      var note = event.detail;
-      resetWrapper();
-      drawNote(note);
-    });
+  var ContentManager = function(){
+    this._wrapper = document.querySelector('#note-content-wrapper');
   }
 
-  function resetWrapper() {
-    _wrapper.innerHTML = '';
-  }
+  ContentManager.prototype = {
+    start() {
+        window.addEventListener('note-open', (function(event) {
```

Comment:
Instead of binding a function as the listener, you should set the argument as `this` and deal with the event in the `handleEvent` method.